chat-client/src/client/chat.ts

```diff
@@ -35,12 +37,13 @@ import {
     TabChangeParams,
     TabRemoveParams,
 } from '@aws/language-server-runtimes-types'
-import { MynahUIDataModel } from '@aws/mynah-ui'
+import { MynahUI, MynahUIDataModel } from '@aws/mynah-ui'
 import { ServerMessage, TELEMETRY, TelemetryParams } from '../contracts/serverContracts'
 import { ENTER_FOCUS, EXIT_FOCUS } from '../contracts/telemetry'
 import { Messager, OutboundChatApi } from './messager'
 import { InboundChatApi, createMynahUi } from './mynahUi'
 import { TabFactory } from './tabs/tabFactory'
+import { MynahUITabStoreModel } from '@aws/mynah-ui/dist/static'
```

Comment:
nit: this type should be exported from MynahUI to avoid import from nested directory.

Reply:
oh yes, i missed the auto import here

Reply:
newer versions of mynah now export this